routes/articles.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express')
+
+const router = express.Router()
+
+router.get('/', (req, res) => {
+    const db = req.app.get('db')
+    db.query('Select * from articles', (err, result) => {
```

Comment:
Is this supposed endpoint supposed to return all articles? Should it not return the most recent 5-10 per page or return by ID?

Reply:
This endpoint would be `/articles`, so it should return articles.
* To get one article, it could be `GET` `/articles/article/:id`.
* To delete an article, it could be `DELETE` `/articles/article/:id`.
* To edit one it could be `POST` `/articles/article/:id`.
* To add an article it could be `PUT` `/articles/article`.

[This](https://restfulapi.net/resource-naming/) is a good read on how to name resources, but it might be worth looking at other articles as well.